Src/FluentAssertions/Execution/AssertionFailedException.cs

```diff
@@ -16,6 +17,7 @@ public AssertionFailedException(string message)
     {
     }
 
+    [ExcludeFromCodeCoverage]
     protected AssertionFailedException(SerializationInfo info, StreamingContext context)
         : base(info, context)
     {
```

Comment:
This was introduced in https://github.com/fluentassertions/fluentassertions/pull/268, but since the corresponding test was removed somewhere around the binary serializer issue, this is neither referenced nor tested against. And since `XmlSerializer` cannot serialize such an `Exception` (throws, because of some `IDictionary` which cannot be serialized), we could even consider to remove it, or leave it with exclude of CC.